### PR TITLE
[config] Update conan, add timeout and disable package id computation cache

### PR DESCRIPTION
**Pull request by danimtb**
Also removing slack entries not needed anymore

### CODE REVIEW

File: .c3i/config_v1.yml [hunk 1]
```diff
@@ -42,6 +42,8 @@ tasks:
       - name: "continuous-integration/jenkins/pr-merge"
       - name: "c3i/conan-v2/pr-merge"
         required_for_references: "conan_v2_ready_references.yml"
+  build_single_reference:
+    package_id_cache_type: "artifactory"
```
Comment (RubenRBS) [inline .c3i/config_v1.yml:46]: So this one is staying? I thought both would be "none" :)

**Review by RubenRBS — APPROVED** [review]

File: .c3i/config_v1.yml (hunk 1 above)
Comment (danimtb) [inline .c3i/config_v1.yml:46]:
```suggestion
  build_single_reference:
    package_id_cache_type: "none"
```

Comment (danimtb) [reply .c3i/config_v1.yml]: Sorry, this was a leftover from other changes. I have changed it to "none" (although it is the default), so we are sure this is the value

**Review by RubenRBS — DISMISSED** [review]

File: .c3i/config_v1.yml [hunk 2]
```diff
@@ -42,6 +42,8 @@ tasks:
       - name: "continuous-integration/jenkins/pr-merge"
       - name: "c3i/conan-v2/pr-merge"
         required_for_references: "conan_v2_ready_references.yml"
+  build_single_reference:
+    package_id_cache_type: "none"
```
Comment (uilianries) [inline .c3i/config_v1.yml:46]:
```suggestion
    package_id_cache_type: "none"
    timeout_minutes: 600
    large_timeout_minutes: 800
    large_timeout_references:
      - boost
      - qt
```

File: .c3i/config_v2.yml [hunk 3]
```diff
@@ -29,6 +24,8 @@ tasks:
   access_request:
       request_issue_url: https://github.com/conan-io/conan-center-index/issues/4
       max_inactivity_days: 0
+  build_single_reference:
+    package_id_cache_type: "none"
```
Comment (uilianries) [inline .c3i/config_v2.yml:28]:
```suggestion
    package_id_cache_type: "none"
    timeout_minutes: 600
    large_timeout_minutes: 800
    large_timeout_references:
      - boost
      - qt
```

**Review by uilianries — COMMENTED** [review]
Just getting a ride here.

**Review by uilianries — APPROVED** [review]

**Review by RubenRBS — APPROVED** [review]

**Pull request merged**